app/models/claim/base_claim.rb

```diff
@@ -285,6 +285,10 @@ def earliest_representation_order
       end.first
     end
 
+    def earliest_representation_order_date
+      earliest_representation_order.try(:representation_order_date)
```

Comment:
do you know in ruby 2.3.1+ you can use &.method as an alternative to .try(:method) 
e.g.
```
earliest_representation_order&.representation_order_date
```